packages/network/lib/index.ts

```diff
@@ -19,3 +19,5 @@ export {
 export { allowDestroy } from './allow-destroy'
 
 export { concatStream } from './concat-stream'
+
+export type { ParsedHost } from './types'
```

Comment:
not sure we need to export these

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/6c80197bea6fbcef901c7bbeda58474657cf8125